sagenb/misc/misc.py

```diff
@@ -30,7 +30,15 @@ def g(*args, **kwds):
 
 min_password_length = 6
 
-import os, cPickle, socket, sys
+import os
+import socket
+import sys
+
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
```

Comment:
I haven't trawled through the sagenb source enough to be sure if it's worth bothering, but if it used the `six` module it would handle stuff like this as well.